projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -33,6 +33,8 @@ export class FirstErrorPipe implements PipeTransform {
       errorMessage = '%FIELDLABEL% is required';
     } else if (keys[0] === 'pattern') {
       errorMessage = 'The data entered is not valid for %FIELDLABEL%';
+    } else if (keys[0] === 'markDownPattern') {
+      errorMessage = 'The data entered is not valid for %FIELDLABEL%. Avoid using Markdown for links i.e [text](url), image and link tags - <img src="" /> or <a href="">text</a>';
```

Comment:
This is too specific - we probably don't want to tell the user exactly what they've gone wrong - see revised message in EXUI-330